03-make/make_simplest/Makefile

```diff
@@ -1,9 +1,11 @@
-
+# blank line is better to be removed
 hello: main.o
 	gcc -o hello main.o -lc
-
-main.o: main.c
+main.o:
+#main.c -removed from 'main.o: main.c' line
 	gcc -c main.c -o main.o
-
 clean:
-	-rm *.o main a.out
+	rm -f *.o hello 
+# a.out - removed from line above, since the is no such file cread afrer make
```

Comment:
It is better to include such comments in commit message.

Reply:
Ok. I'm not good experienced in Git. Thanks a lot for advise.

Reply:
@yekovalyov is this good?